src/wp-includes/class-wp-theme-json.php

```diff
@@ -526,60 +636,104 @@ public function get_stylesheet( $type = 'all' ) {
 	 *     style-property-one: value;
 	 *   }
 	 *
-	 * Additionally, it'll also create new rulesets
-	 * as classes for each preset value such as:
-	 *
-	 *     .has-value-color {
-	 *       color: value;
-	 *     }
-	 *
-	 *     .has-value-background-color {
-	 *       background-color: value;
-	 *     }
-	 *
-	 *     .has-value-font-size {
-	 *       font-size: value;
-	 *     }
-	 *
-	 *     .has-value-gradient-background {
-	 *       background: value;
-	 *     }
-	 *
-	 *     p.has-value-gradient-background {
-	 *       background: value;
-	 *     }
-	 *
 	 * @since 5.8.0
+	 * @since 5.9.0 Renamed to get_block_classes and no longer returns preset classes.
 	 *
-	 * @param array $style_nodes   Nodes with styles.
-	 * @param array $setting_nodes Nodes with settings.
+	 * @param array $style_nodes Nodes with styles.
 	 * @return string The new stylesheet.
 	 */
-	private function get_block_styles( $style_nodes, $setting_nodes ) {
```

Comment:
Is this method being removed? This is breaking change.

Reply:
It's a private method within a private core-only class. Removing it should be okay.